receivers/telegram/config.go

```diff
@@ -19,6 +19,7 @@ var SupportedParseMode = map[string]string{"Markdown": "Markdown", "MarkdownV2":
 type Config struct {
 	BotToken              string `json:"bottoken,omitempty" yaml:"bottoken,omitempty"`
 	ChatID                string `json:"chatid,omitempty" yaml:"chatid,omitempty"`
+	MessageThreadID       string `json:"message_thread_id,omitempty" yaml:"message_thread_id,omitempty"`
```

Comment:
according to the [documentation](https://core.telegram.org/bots/api#sendmessage) the field is of integer type. Here we use string, which is OK. However, I would recommend adding validation in the NewConfig function that would check that the string is a valid integer. 
This is important because the wrong configuration can lead to non-deliverable notifications.

Reply:
Please add positive and negative tests for the new field.

Reply:
I added tests @yuri-tceretian. Can you please have a look and let me know if they are OK? 💐